.editorconfig

```diff
@@ -0,0 +1,18 @@
+# editorconfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+max_line_length = 120
```

Comment:
thats for everything else than python, actually we should add the python rule at some point